src/Worker/Grpc/GrpcDurableTaskWorker.Processor.cs

```diff
@@ -1,523 +1,524 @@
+        }
+
+        ILogger Logger => this.worker.logger;
+
+        public async Task ExecuteAsync(CancellationToken cancellation)
+        {
+            while (!cancellation.IsCancellationRequested)
+            {
+                try
+                {
+                    AsyncServerStreamingCall<P.WorkItem> stream = await this.ConnectAsync(cancellation);
+                    await this.ProcessWorkItemsAsync(stream, cancellation);
+                }
+                catch (RpcException) when (cancellation.IsCancellationRequested)
+                {
+                    // Worker is shutting down - let the method exit gracefully
+                    break;
+                }
+                catch (RpcException ex) when (ex.StatusCode == StatusCode.Cancelled)
+                {
+                    // Sidecar is shutting down - retry
+                    this.Logger.SidecarDisconnected();
+                }
+                catch (RpcException ex) when (ex.StatusCode == StatusCode.Unavailable)
+                {
+                    // Sidecar is down - keep retrying
+                    this.Logger.SidecarUnavailable();
+                }
+                catch (OperationCanceledException) when (cancellation.IsCancellationRequested)
+                {
+                    // Shutting down, lets exit gracefully.
+                    break;
+                }
+                catch (Exception ex)
+                {
+                    // Unknown failure - retry?
+                    this.Logger.UnexpectedError(ex, string.Empty);
+                }
+
+                try
+                {
+                    // CONSIDER: Exponential backoff
+                    await Task.Delay(TimeSpan.FromSeconds(5), cancellation);
+                }
+                catch (OperationCanceledException) when (cancellation.IsCancellationRequested)
+                {
+                    // Worker is shutting down - let the method exit gracefully
+                    break;
+                }
+            }
+        }
+
+        static string GetActionsListForLogging(IReadOnlyList<P.OrchestratorAction> actions)
+        {
+            if (actions.Count == 0)
+            {
+                return string.Empty;
+            }
+            else if (actions.Count == 1)
+            {
+                return actions[0].OrchestratorActionTypeCase.ToString();
+            }
+            else
+            {
+                // Returns something like "ScheduleTask x5, CreateTimer x1,..."
+                return string.Join(", ", actions
+                    .GroupBy(a => a.OrchestratorActionTypeCase)
+                    .Select(group => $"{group.Key} x{group.Count()}"));
+            }
+        }
+
+        async ValueTask<OrchestrationRuntimeState> BuildRuntimeStateAsync(
+            P.OrchestratorRequest orchestratorRequest,
+            ProtoUtils.EntityConversionState? entityConversionState,
+            CancellationToken cancellation)
+        {
+

```

Comment:
Can this whitespace-only change be reverted?

Reply:
fixed